documentation/asciidoc/computers/config_txt/conditional.adoc

```diff
@@ -66,7 +66,7 @@ These are particularly useful for defining different `kernel`, `initramfs`, and
 
 Remember to use the `[all]` filter at the end, so that any subsequent settings aren't limited to Raspberry Pi 2 hardware only.
 
-It is important to note that the Raspberry Pi Zero W will see the contents of `[pi0w]` AND `[pi0]`. Likewise, a Raspberry Pi 3B+ sees `[pi3+]` AND `[pi3]`, and a Raspberry Pi 400 sees `[pi400]` AND `[pi4]`. If you want a setting to apply only to Raspberry Pi Zero, Raspberry Pi 3B or Raspberry Pi 4B, you need to follow it (order is important) with a setting in the `[pi0w]`, `[pi3+]` or `[pi400]` section that reverts it.
+NOTE: A Raspberry Pi Zero W will see the contents of `[pi0w]` *and* `[pi0]`. Likewise, Raspberry Pi 3B+ see `[pi3+]` *and* `[pi3]` sections, Raspberry Pi 400 see `[pi400]` *and* `[pi4]` sections, Raspberry Pi Compute Module 4 see `[cm4]` *and* `[pi4]` sections and Raspberry Pi Compute Module 4S see `[cm4s]` *and* `[pi4]` sections. If you want a setting to apply _only_ to a Raspberry Pi Zero, Raspberry Pi 3B, or Raspberry Pi 4B then you need to follow it - the ordering is significant - with a setting in a `[pi0w]`, `[pi3+]`, `[pi400]`, `[cm4]` or `[cm4s]` section which reverts the shared setting.
```

Comment:
**Perhaps** it's worth replacing this overly-verbose note with something shorter like:
"NOTE: Some models of Raspberry Pi (Zero W, Pi 3B+, Pi 400, CM4, etc.) see the settings for _multiple_ filters (see the details in the table above), so if you e.g. want a setting to _only_ apply to a Raspberry Pi 4B without also applying to a Raspberry Pi 400, the setting in a `[pi4]` section would need to be reverted by a later setting in a `[pi400]` section - the ordering of the sections is significant. Alternatively, you could use a `[board-type=0x11]` filter.

Reply:
Yeah, that's _much_ better! 😅